src/app/pages/task-calendar/cloudsync/cloudsync-form/cloudsync-form.component.ts

```diff
@@ -598,7 +598,7 @@ export class CloudsyncFormComponent {
 
     this.folder_field = _.find(entityForm.fieldConfig, { "name": "folder"}); 
     this.formGroup.controls['credentials'].valueChanges.subscribe((res)=>{
-      if(_.isNil(res)) {
+      if(_.isNil(res) || res === 'null_value') {
```

Comment:
Is it applicable here?
https://github.com/truenas/webui/blob/145133f5a52cd23ff19557a6a0718712d3876887/src/app/pages/common/entity/utils.ts#L5

Reply:
The check wasn't needed at all. Updated.